src/crypto/generichash/mod.rs

```diff
@@ -108,6 +108,16 @@ mod test {
     #[cfg(not(feature = "std"))]
     use prelude::*;
 
+    #[test]
+    fn test_crypto_generichash_state_alignment() {
+        // bindgen does not properly compute the alignment of crypto_generichash_state,
+        // which is why gen.sh in libsodium-sys applies a patch after it has run rust-bindgen
```

Comment:
That should be a comment for gen.sh =)

The comment inside this test should correspond to libsodium docs where it is required for crypto_generichash_blake2b_state to be 64-aligned with the link to libsodium's docs